src/main/java/org/cryptimeleon/craco/sig/sps/kpw15/SPSKPW15VerificationKey.java

```diff
@@ -72,27 +67,15 @@ public SPSKPW15VerificationKey(GroupElement[] C0, GroupElement[] C1,
     }
 
 
-
-
     public GroupElement[] getC0() {
         return C0;
     }
 
-    public void setC0(GroupElement[] c0) {
-        C0 = c0;
-    }
-
     public GroupElement[] getC1() {
         return C1;
     }
 
-    public void setC1(GroupElement[] c1) {
-        C1 = c1;
-    }
-
-    public GroupElement[] getC() {
-        return C;
-    }
+    public GroupElement[] getC() { return C; }
```

Comment:
Same JavaDoc note as for the secret key please.

Reply:
[50cd892](https://github.com/Colamaleon/craco/pull/2/commits/50cd8925bfec3139f3e0de29bc1c3bd81c5f86d5)